install/prereqs/centos.rst

```diff
@@ -4,47 +4,59 @@
 RedHat / CentOS system prerequisites
 ####################################
 
+..   yum install bison curl blas bzip2-devel bzip2 flex fontconfig \
+       freetype-devel gcc-c++ gcc-gfortran git libuuid-devel \
+       libXext libXrender libXt-devel make openssl-devel patch perl \
+       readline-devel tar zlib-devel ncurses-devel cmake glib2-devel \
+       java-1.8.0-openjdk gettext perl-ExtUtils-MakeMaker \
+       centos-release-scl devtoolset-6-gcc devtoolset-6-gcc-c++ \
+       devtoolset-6-gcc-gfortran mesa-libGL-devel
+
+First install the packages required to build the ditribution products:
+
 .. code-block:: bash
 
    yum install bison curl blas bzip2-devel bzip2 flex fontconfig \
        freetype-devel gcc-c++ gcc-gfortran git libuuid-devel \
        libXext libXrender libXt-devel make openssl-devel patch perl \
        readline-devel tar zlib-devel ncurses-devel cmake glib2-devel \
-       java-1.8.0-openjdk gettext perl-ExtUtils-MakeMaker
+       java-1.8.0-openjdk gettext perl-ExtUtils-MakeMaker \
+       mesa-libGL
 
 .. from https://github.com/lsst-sqre/puppet-lsststack/blob/master/manifests/params.pp
 
 Prefix the :command:`yum` command with :command:`sudo` if necessary.
 
-.. note::
+Then install the CentOS Linux Software Collections release file:
 
-   **New since 11.0**: The minimum gcc version required to compile the LSST Science Pipelines is **gcc 4.8.**
-   If you are using our previous factory platform, RedHat/CentOS 6, and you are unable to upgrade to version 7 (which comes with gcc 4.8 as default) consult :ref:`the section below on upgrading compilers in legacy Linux <source-install-redhat-legacy>`.
+.. code-block:: bash
 
-.. _source-install-redhat-legacy:
+   yum install centos-release-scl 
 
-Upgrading compilers for legacy RedHat / CentOS 6
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+and finally the latest compiler packages:
```

Comment:
Should this be a capital “A”? Punctuation in text with lots of “:”s introducing code examples is hard, but I think it should be to match “Then” above and ’’Activate” and “Check” below.